http-server/src/main/java/io/micronaut/http/server/util/localeresolution/CompositeHttpLocaleResolver.java

```diff
@@ -50,7 +60,7 @@ public CompositeHttpLocaleResolver(HttpLocaleResolver[] localeResolvers,
     }
 
     @Override
-    public Locale resolveOrDefault(@NonNull HttpRequest<?> request) {
-        return resolve(request).orElse(defaultLocale);
+    public int getOrder() {
+        return Ordered.LOWEST_PRECEDENCE;
```

Comment:
Setting an order here doesn't make sense because this is the primary bean

Reply:
But if you inject every `HttpLocaleResolver` with `Collection<LocaleResolver>` and you sort the
 resolver,  you would want the `CompositeHttpLocaleResolver` to have the lowest order. Specially, since we don't provide a way to disable `CompositeHttpLocaleResolver`.

Reply:
There is no use case to inject all locale resolvers outside of the composite one. The composite one here is specific to HTTP so there shouldn't be a need to disable it.

Reply:
I've removed the `getOrder`override. I still think it is more correct that `CompositeHttpLocaleResolver`
has the lowest order in a list of `LocaleResolver`,